common/core/desktop/src/kmx/kmx_xstring.cpp

```diff
@@ -173,43 +175,19 @@ PKMX_WCHAR km::kbp::kmx::decxstr(PKMX_WCHAR p, PKMX_WCHAR pStart)
   {
     return p-1;
   }
+
   else if(*(p-1) == UC_SENTINEL) return p-1;
```

Comment:
I wonder if that ever will be a problem since we want to end up at the next UC_SENTINEL to the left if possible?
- For "normal" characters (u"\uFFFFa") or Surrogate Pairs we move 1 to the left to UC_SENTINEL.
- For all CODE__xxx where (CODE__SIZE[*(q + 1)] + 1 == i) we jump directly to next UC_SENTINEL to the left.
Are there any cases where we would run into problems when we are one position right to a UC_SENTINEL and we are then moving 1 step to the left to that UC_SENTINEL?